### ISSUE TITLE
Can a Powerstream charge an Extra Battery from the AC grid?

### ISSUE THREAD

**Issue by degerrit** [svenerbe/ecoflow_dynamic_power_adjustment#18]
Sorry to abuse an issue for this, but this seems like a knowledgeable crowd with a similar use-case than I have in mind (I'd also adapt the power from a Powerstream and extra battery dynamically based on live household consumption).

But I would also like to charge the battery based on my grid injection from AC solar. Is this possible?
I have no direct DC power to attach to a PS (rooftop voltage would be way too high).

If I read the specifications, I guess this could be the equivalent of setting WN511_SET_PERMANENT_WATTS_PACK to a negative value, unless there are other API endpoints for that?

(I also have a Delta 2 that I'm using for this purpose w/o PS, but adding a PS to a D2 would make the EB obsolete, so I'm thinking of setting up a PS with the EB separately)

**Comment by Keff789** [comment]
I dont think it is possible to charge a battery over the Powerstream, so not really in the scope of this script.

**Comment by rgerhards** [comment]
I am not currently using this script, but I have last year experimented with a Delta 2 and a Shelly plug. If I turn on the shelly plug grid wise, the Delta 2 did load. Just to give you a rough idea of what you can do.
